### PR TITLE
Have optional fields in Timeline use space rather than hyphens

### PR DESCRIPTION
**Pull request by annebyrne**

### CODE REVIEW

File: public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js [hunk 1]
```diff
@@ -24,9 +24,9 @@ export class TimelineItem extends React.Component {
 
   render () {
     const value = this.props.fieldValue || {
-      title: "-",
+      title: " ",
```
Comment (shaundillon) [inline public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js]: I'm not sure this is valid in Dynamo, which is why a dash was used.

Comment (annebyrne) [reply public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js]: Gaaaaaaaah 😱 Thanks for spotting. I'll take a look.

Comment (annebyrne) [reply public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js]: So from messing around with the `CODE` & `DEV` DynamoDB tables it seems to accept the spaces. Do you know where this was covered in a PR or anywhere else?

Comment (shaundillon) [reply public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js]: @susiecoleman and @philmcmahon Any idea? ^

Comment (tomrf1) [reply public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js]: Dynamo doesn't allow an empty string...not sure if a space will make it in

Comment (annebyrne) [reply public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js]: It seems to work from modifying this atom directly in Dynamo: https://atomworkshop.code.dev-gutools.co.uk/atoms/timeline/07a75b03-92fd-4c78-9052-ccbb25c478d4/edit

What do you think @shaundillon ?

Comment (shaundillon) [reply public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js]: Ahh, that's what I was thinking. No idea why a dash was chosen over a space then. Maybe for visibility?

Comment (tomrf1) [reply public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js]: 💯 much better than the dash!

**Comment by annebyrne** [comment]
This seems to be working in `CODE` so I am going to go ahead and ship if that's OK? @tomrf1 @shaundillon

**Comment by shaundillon** [comment]
Yep, s'all good with me

**Comment by tomrf1** [comment]
👍

**Pull request merged**